src/pages/embedding/public/index.tsx

```diff
@@ -147,6 +140,7 @@ export default function PublicEmbeddingPage(): ReactElement {
 
         <PublicIFrame
           src="/question/88bbf0d1-8982-47a9-9914-dbc29af6b36c"
+          key={appStylesKey}
```

Comment:
I'd rather fix the Metabase itself to allow changing fonts on the fly. I notice every time we change any static embed settings in Metabase, we always refresh the iframe. This would be worse when questions aren't cached and take some time to load since users should be able to toggle different knobs without having to reload the question again.

This is true for edumation public embed. Previously, the iframe doesn't refresh when changing parameters, but with this PR it will always refresh which looks like a step backwards in my opinion.

We could fix this in Metabase by adding this block

```ts
setOptions({
        ...parseSearchOptions(window.location.search),
        ...parseHashOptions(window.location.hash),
      })
```
to `frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx`

since that's exactly how embedding was initialized.

https://github.com/metabase/metabase/blob/0bb85bf4aef2fe0dd64a3b6ddcbfd11f31460aa1/frontend/src/metabase/lib/embed.js#L44-L49

Embedding options in redux store was never changed for public and static embeds, but for the sake of better preview I think it's fine to allow updating embedding options state.

What do you think?

Reply:
That sounds reasonable to me.